### PR TITLE
Update it.po

### PR DESCRIPTION
**Pull request by albanobattistella**

### CODE REVIEW

File: po/it.po [hunk 1]
```diff
@@ -440,68 +442,59 @@ msgid "Quit"
...
 msgid "Toggle Reverse Order"
-msgstr ""
+msgstr "Attiva/disattiva ordine inverso"
 
 #: src/shortcuts_dialog.ui:85
-#, fuzzy
-msgid "Toggle Widgets Style"
-msgstr "Stile widget"
+msgid "Change Widgets Style"
+msgstr "Cambia lo stile widget"
```
Comment (fsobolev) [inline po/it.po]: Please translate "**Toggle** Widgets Style" here, this is recent change, sorry for inconvenience

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: po/it.po (hunk 1 above)
Comment (albanobattistella) [reply po/it.po]: Attiva/disattiva lo stile dei widget

Comment (fsobolev) [reply po/it.po]: Idk how it should be in Italian, the thing is you have old `msgid` there ("Change...", but I recently changed it to "Toggle...")

**Review by fsobolev — APPROVED** [review]

**Comment by fsobolev** [comment]
Thank you! 😊

**Pull request merged**